compose.yml

```diff
@@ -11,7 +11,11 @@ services:
       - ./testmcserver:/testmcserver
     environment:
       - MINECRAFT_VERSION=${MINECRAFT_VERSION}
+      - OPS_UUID=${OPS_UUID}
+      - OPS_NAME=${OPS_NAME}
+      - OPS_LEVEL=${OPS_LEVEL}
```

Comment:
"OPS" implies the capability to have multiple operators but the script only supports a single operator. The environment variables should be updated accordingly.

Reply:
Addressed